src/DependencyInjection/HappyrAuth0Extension.php

```diff
@@ -39,35 +35,24 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
 
-        if (!$config['firewall']['enabled']) {
-            $container->removeDefinition(Auth0Authenticator::class);
-        } else {
-            $entryPointDefinition = $container->getDefinition(Auth0EntryPoint::class);
-            $entryPointDefinition->replaceArgument(2, $config['config']['clientId']);
-            $entryPointDefinition->replaceArgument(3, $config['login_domain'] ?? $config['config']['domain']);
-            $entryPointDefinition->replaceArgument(4, $config['config']['scope']);
-            $this->configureFirewall($container, $config['firewall']);
-        }
+        $this->configureSdkConfiguration($container, $config['sdk'] ?? []);
 
-        if (null === $config['config']['httpRequestFactory']) {
-            $config['config']['httpRequestFactory'] = new Reference(RequestFactoryInterface::class, ContainerInterface::NULL_ON_INVALID_REFERENCE);
-        }
-        if (null === $config['config']['httpResponseFactory']) {
-            $config['config']['httpResponseFactory'] = new Reference(ResponseFactoryInterface::class, ContainerInterface::NULL_ON_INVALID_REFERENCE);
-        }
-        if (null === $config['config']['httpStreamFactory']) {
-            $config['config']['httpStreamFactory'] = new Reference(StreamFactoryInterface::class, ContainerInterface::NULL_ON_INVALID_REFERENCE);
+        if ($config['firewall']['enabled']) {
```

Comment:
Dont forget to remove `Auth0Authenticator` service if it is not enabled.

Reply:
done, removed also Auth0Entrypoint